modules/planning/tasks/deciders/decider.cc

```diff
@@ -36,5 +40,58 @@ apollo::common::Status Decider::Execute(Frame* frame) {
   return Process(frame);
 }
 
+
+bool Decider::BuildStopDecision(
+    const std::string& stop_wall_id, const double stop_line_s,
+    const double stop_distance, const StopReasonCode& stop_reason_code,
+    const std::vector<std::string>& wait_for_obstacles, Frame* const frame,
+    ReferenceLineInfo* const reference_line_info) {
+  CHECK_NOTNULL(frame);
+  CHECK_NOTNULL(reference_line_info);
+
+  // check
+  const auto& reference_line = reference_line_info->reference_line();
+  if (!WithinBound(0.0, reference_line.Length(), stop_line_s)) {
+    AERROR << "stop_line_s[" << stop_line_s << "] is not on reference line";
+    return 0;
```

Comment:
Better to return true or false to avoid confusion. I blame the original author of the code :-)

Reply:
that would be me.  will fix it. thx